examples/aws/terraform/ha-autoscale-cluster/vars.tf

```
@@ -240,3 +240,30 @@ variable "teleport_auth_type" {
   type    = string
   default = "local"
 }
+
+// (optional) Change the default tags applied to all resources.
+variable "default_tags" {
+  type    = map(string)
+  default = {}
+}
+
+// Whether to trigger instance refresh rollout for Teleport Auth servers when
+// the launch template or resource tags change.
+variable "enable_auth_asg_instance_refresh" {
+  type    = bool
+  default = true
```

```suggestion
  default = false
```

As discussed we shouldn't automatically kick off instance refreshes as it'd be a breaking change for folks currently using the module who schedule their refreshes manually during maintenance windows. An unfortunate fact of not having versioned this module :(